### DISCUSSION TITLE
Is there a difference between Windows 10 and Windows 11 for AI?

### DISCUSSION THREAD

**Discussion by rengecky** [oobabooga/text-generation-webui#2402] [Q&A]
Nvidia has released AI support in driver 532.03 for Windows 11. As a normal user, does it make sense for me to upgrade from Windows 10 to Windows 11 or is there no difference in performance?

**Comment by BetaDoggo** [comment]
This will make no difference unless the project you are using is DirectML based, which very few are. Even then I'm not sure how much of an uplift this is, since DirectML was slower than the PyTorch implementations before the update.
